### PR TITLE
feat: add oidc4vp support to create verifier profile.

### PR DESCRIPTION
**Pull request by vkubiv**
Signed-off-by: Volodymyr Kubiv <volodymyr.kubiv@euristiq.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -62,6 +62,7 @@ require (
 	github.com/go-openapi/jsonpointer v0.19.5 // indirect
 	github.com/go-openapi/swag v0.21.1 // indirect
 	github.com/go-sql-driver/mysql v1.6.0 // indirect
+	github.com/go-test/deep v1.0.8 // indirect
```
Comment (mishasizov-SK) [inline go.mod:65]: Do we have any company restrictions to import 3rd party packages?

Comment (vkubiv) [reply go.mod:65]: It is used just for tests. So should be any proplems

File: pkg/restapi/v1/verifier/controller_test.go [hunk 2]
```diff
@@ -43,22 +52,33 @@ var (
...
+	kms.ECDSAP384TypeDER,
+	kms.RSAPS256Type,
+	kms.BLS12381G2Type,
+}
+
 //nolint:gochecknoglobals
 var (
 	verificationChecks = &verifiersvc.VerificationChecks{
-		Credential: &verifiersvc.CredentialChecks{
+		Credential: verifiersvc.CredentialChecks{
```
Comment (aholovko) [inline pkg/restapi/v1/verifier/controller_test.go:69]: is there a specific reason why Credential is value type while Presentation below is reference?

File: pkg/restapi/v1/verifier/openapi.gen.go [hunk 3]
```diff
@@ -9,42 +9,53 @@ import (
...
 	// URI of the verifier.
 	Url *string `json:"url,omitempty"`
 }
 
+// Model for OIDC4VPConfig configuration.
+type OIDC4VPConfig struct {
+	// DID method of the DID to be used for signing.
+	DidMethod externalRef0.DIDMethod `json:"didMethod"`
+
+	// Cryptographic algorithms used to sign oipd vp request object.
```
Comment (aholovko) [inline pkg/restapi/v1/verifier/openapi.gen.go:42]: oipd -> oidc?

do we need `Ro` prefix in `RoSigningAlgorithm`? if this is important to emphasize "request object" maybe name it something like "RequestSigningAlgorithm" ("ro" has a collocation with "read-only")

**Review by aholovko — APPROVED** [review]

File: pkg/restapi/v1/verifier/controller_test.go (hunk 2 above)
Comment (vkubiv) [reply pkg/restapi/v1/verifier/controller_test.go:69]: Credential is not optional, Presentation is optional

File: pkg/restapi/v1/verifier/openapi.gen.go (hunk 3 above)
Comment (vkubiv) [reply pkg/restapi/v1/verifier/openapi.gen.go:42]: "Request" is confusing, but yes "ro" is definitely not the best name. 

I will include fix for oipd and "ro" in next PR

**Review by mishasizov-SK — APPROVED** [review]

**Pull request merged**